### ISSUE TITLE
Tensorflow Inference over satellite image tiles

### ISSUE THREAD

**Issue by Geo-Trackers** [developmentseed/label-maker#126]
Hello Everyone, 

I want to detect brick-kilns in satellite imagery. I trained my model using Tensorflow object detection API. Now, I want to run inference over my satellite image tiles. I want to  use gdaltools to generate image tiles and run inference over each tiles, and stich them back together for visualization over the large area of satellite imagery? Also, if my brick kiln falls in two image tiles, will the script tf_od_predict.py compute non-maximum suppression and get rid of low probability predictions, while i stich the images tiles back together.
Thank you. I look forward to hear from you.

**Comment by Geoyi** [comment]
You're right here, @Geo-Trackers.  However, a large object may cross two tiles or more tiles so can write a utility function to get rid of bounding boxes that overlap by 75% threshold for instance. But I will recommend you to test with different threshold base on your case and object.
